File: Java/Answer7.java
class Answer7 {
    public static void  main(String[] args) {
        int targetInt = 2344;
        int revTargetInt = 0;
        int remainder = 0;

        String targeStr = "lollipop";
        String revTargetStr = "";
        int lenTargetSize = targeStr.length();
        System.out.println(lenTargetSize);
        for (int i = lenTargetSize-1; i>=0; i--) {
            revTargetStr += targeStr.charAt(i);
        }
        System.out.println("Target String : "+targeStr);
        System.out.println("Reverse Integer :"+revTargetStr);

        System.out.println("Target Interger : "+targetInt);
        while(targetInt>0) {
            remainder = targetInt%10;
            revTargetInt = (revTargetInt*10)+remainder;
            targetInt = targetInt/10;
        }
        System.out.println("Reverse Integer : "+revTargetInt);

    }
}

File: C++/Answer7.cpp
#include<iostream>
using namespace std;

int main () {
    int targetInt = 3425;
    int revTargetInt = 0;
    int remainder = 0;
    
    string targetStr = "honey";
    string revTargetStr = "";
    int lenTargetSize = targetStr.size();
    for (int i = lenTargetSize-1 ; i >=0 ; i--) {
        revTargetStr += targetStr.at(i);
    }
    std::cout << "Target String : " << targetStr << std::endl;
    std::cout << "Reverse Integer : " << revTargetStr << std::endl;

    std::cout << "Targer Integer : " << targetInt << std::endl;
    while(targetInt>0) {
        remainder = targetInt % 10;
        revTargetInt = (revTargetInt*10)+ remainder;
        targetInt = targetInt / 10;
    }
    std::cout << "Reverse Integer : " << revTargetInt << std::endl;
    

    return 0;
}